commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/PaperCommandRegistration.java

```diff
@@ -38,6 +89,18 @@ public boolean isBukkitCommand(CommandNode<Source> node) {
 		return isBukkitCommand.test(node);
 	}
 
+	@SuppressWarnings("unchecked")
+	public CommandDispatcher<Source> getPaperDispatcher() {
+		CommandDispatcher<?> commandDispatcher;
+		try {
+			commandDispatcher = (CommandDispatcher<?>) dispatcherField.get(paperCommandsInstance);
+		} catch (IllegalAccessException e) {
+			// This doesn't happen
+			commandDispatcher = null;
+		}
+		return (CommandDispatcher<Source>) commandDispatcher;
+	}
```

Comment:
Could return a `CommandDispatcher<Source>` directly here:

```suggestion
	@SuppressWarnings("unchecked")
	public CommandDispatcher<Source> getPaperDispatcher() {
		try {
			return (CommandDispatcher<Source>) dispatcherField.get(paperCommandsInstance);
		} catch (IllegalAccessException e) {
			// This doesn't happen
			return null;
		}
	}
```